robottelo.properties.sample

```diff
@@ -127,3 +127,13 @@ test.virtual_machines_list=127.0.0.1,127.0.0.1,127.0.0.1,127.0.0.1,127.0.0.1
 # Class `ConcurrentTestCase` and its subclasses use this setting when
 # computing statistics of each performance test case, grouped in buckets.
 csv.num_buckets=10
+
+# Target repository names to be synchronized by Pulp.
+test.target_repos=Repo Example 1,Repo-Example-2
```

Comment:
Can you do one of these?

    test.target_repos=
    # test.target_repos=

If the user must provide a value for some field and the user fails to provide a value, then we want an error to be thrown as soon as reasonably possible. Providing a bogus default value runs counter to that goal.

In other words:

* If a value must be provided and we can give a sane default, give a sane default.
* If a value must be provided and we cannot give a sane deafult, don't give an insane default.